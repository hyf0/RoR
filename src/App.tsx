import { FC } from 'react'

const App: FC = () => {
  return <div>hello, world</div>
}

export default App
