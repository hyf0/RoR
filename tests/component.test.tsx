import { render } from '@testing-library/react'
import { describe, it, expect } from 'vitest'
import App from '~/src/App'

describe(`App.tsx`, () => {
  it('should mounted without errors', async () => {
    // ARRANGE
    const result = render(<App />)

    // ACT
    // noop

    // ASSERT
    expect(result.baseElement).toMatchSnapshot()
  })
})
