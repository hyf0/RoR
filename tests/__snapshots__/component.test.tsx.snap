// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App.tsx > should mounted without errors 1`] = `
<body>
  <div>
    <div>
      hello, world
    </div>
  </div>
</body>
`;
